meshtastic/tests/test_powermon_power_supply.py

```
@@ -159,3 +161,52 @@ def test_powermon_public_exports_remain_available() -> None:
+    def _fake_import_module(
+        name: str, package: str | None = None
+    ) -> types.ModuleType:
+        if package == "meshtastic.powermon" and name in (".ppk2", ".riden"):
+            missing = ModuleNotFoundError("optional backend missing")
+            missing.name = "riden" if name == ".riden" else "ppk2_api"
+            raise missing
+        return real_import_module(name, package)
+
+    monkeypatch.setattr(importlib, "import_module", _fake_import_module)
+
+    backend_cls = getattr(powermon, "RidenPowerSupply")
```

Ruff (0.15.4)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)